pkg/encoding/encoding.go

```diff
@@ -154,8 +154,8 @@ func tryFindProto(in []byte) ([]byte, bool) {
 
 const jsonStartChars = "{["
 
-// TryFindJson searches for the start of a valid json substring, and, if found, returns the json.
-func tryFindJson(in []byte) (*json.RawMessage, bool) {
+// TryFindJSON searches for the start of a valid json substring, and, if found, returns the json.
```

Comment:
This should be lower-case.

```suggestion
// tryFindJSON searches for the start of a valid json substring, and, if found, returns the json.
```